lesson_09/types/types_app/src/main/java/com/codedifferently/lesson9/dataprovider/DavidSmithProvider.java

```diff
@@ -0,0 +1,23 @@
+package com.codedifferently.lesson9.dataprovider;
+
+import java.util.Map;
+
```

Comment:
The linter is complaining about this extra space, which should've been fixed with `./gradlew spotlessApply`.